docs/api/1.x/_details-patch-object.rst

```diff
@@ -92,3 +87,17 @@ on permission operations and can be omitted.
         { "op": "add", "path": "/permissions/read/system.Everyone" },
         { "op": "remove", "path": "/permissions/read/fxa:bob" }
     ]
+
+3. On JSON Merge operations, ``null`` values can be used to remove all principals
+   from a specific permission level. As a consequence, using ``null`` for permissions
+   on other types of patch operations is considered a no-op.
```

Comment:
Wait, maybe I misunderstood, I didn't think that was a consequence. I would have written something like `However, using null instead of a set of principals when using other patch types is not valid and will result in an error`.

Reply:
From my understanding what I've described is the implemented behavior. This is what I've tried to explain in the PR by saying "As a side effect this allows passing null to regular patch operations with no effect".

Reply:
Ah, yes, then I misunderstood. I thought we had logic to only do the preprocess/postprocess null step if we were in a merge patch. In that case, why do we have this preprocess/postprocess step, versus just allowing permissions to be nullable? I'm guessing it's because we want to only allow nulls in certain kinds of operations, but it seems like we added it to the PermissionSchema, so it seems like it's global anyhow?